apis/filesystem/src/main/java/org/jclouds/filesystem/util/Utils.java

```diff
@@ -49,7 +55,23 @@ public static void deleteRecursively(File file) throws IOException {
             }
          }
       }
-      Files.delete(file.toPath());
+
```

Comment:
This doesn't seem right -- usually a virus checker or similar is holding open a file descriptor if you cannot immediately delete on Windows.